CHANGELOG.md

```diff
@@ -42,14 +44,49 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+ * runc has been updated to OCI runtime-spec 1.2.0, and supports all Linux
+   features with a few minor exceptions. See `docs/spec-conformance.md` for
```

Comment:
nit: should this be a link? I.e.

```suggestion
   features with a few minor exceptions. See [docs/spec-conformance.md](docs/spec-conformance.md) for
```

Reply:
I don't think that link will work when I copy it to the release notes, but sure.

Reply:
I guess we can use absolute path.